pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -22,12 +25,14 @@ stdenv.mkDerivation rec {
   };
 
   unpackPhase = "tar xvjf $src; cd lsof_*; tar xvf lsof_*.tar; sourceRoot=$( echo lsof_*/); ";
+  
+  patches = [ ./dfile.patch ];
 
-  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;';";
+  preBuild = "sed -i Makefile -e 's/^CFGF=/&	-DHASIPv6=1/;' -e s/-lcurses/-lncurses/ ;";
 
   configurePhase = ''
     # Stop build scripts from searching global include paths
-    export LSOF_INCLUDE=/$(md5sum <(echo $name) | awk '{print $1}')
+    # export LSOF_INCLUDE=/$(md5sum <(echo $name) | awk '{print $1}')
```

Comment:
I did this one; we could just direct it to `/dev/null` (the `md5sum` call is just meant to create a file that we're confident doesn't exist). Basically it's to stop lsof from importing global system paths. And yes, we'll want to keep it unless we get around that some other way.